torch/csrc/jit/tensorexpr/constant_folder.h

```diff
@@ -58,6 +166,14 @@ class ConstantFolder : public IRMutator {
     return mutateBinaryOp(v, this, v->propagate_nans());
   }
 
+  const Expr* mutate(const Cast* v) override {
+    if (v->src_value()->isConstant()) {
+      return evaluateOp(v);
```

Comment:
We probably should try to `evaluateOp` all ops if all operands are constant, not just `Cast`, but I couldn't find where we do this for, say, `Mul`.